docs/docs/v4-migration.md

```diff
@@ -46,7 +46,8 @@ instead of `null`
 
 ### Typescript Deep Imports
 
-If you were using deep imports from RNTP, the `src` has been completely
+1. If you were using deep imports from RNTP, the `src` has been completely
 reorganized, and so you may need to adjust your imports accordingly. If you've
 been importing everything directly (ex. `import ... from 'react-native-track-player';`)
 then you don't need to do anything.
+1. The `PlaybackStateEvent` interface has been renamed to `PlaybackState`
```

Comment:
Can we mention that this is TS only?

Reply:
This point and the one above it are under the `### Typescript Deep Imports` header. Which maybe could just be renamed to Typescript Imports. Thoughts?

Reply:
Updated heading to `Typescript Imports`